.nvmrc

```diff
@@ -1 +1 @@
-8.x
\ No newline at end of file
+14.x
```

Comment:
I think, in the interest of broader compatibility, it would make sense to use a dev node version of 12, test against that, and require that in the package.json. Thought?

Reply:
Maybe not, but 12 isn't EOL yet and I feel like I don't see many modules requiring 14 yet.

Reply:
I mostly moved it to 14 because we have done that in Kibana, but happy to loosen it.

Reply:
I'm under the impression that this is used outside Kibana, justifying the separate package, but I'm not certain.

Reply:
@spalger It appears not to be used based on NPM data, which is not a perfect representation of usage, but my assumption is that download numbers here are mostly based on kibana installation: https://www.npmjs.com/package/tinymath

Reply:
Alright, lets give 14 a shot and see if we hear any complaints.